README.md

```diff
@@ -103,7 +103,7 @@ Step 6: Check Running The Site At  <a href="http://localhost:8083/onlinebookstor
 
 Step 7: Default Username And Password For Admin Is "Admin" And "Admin"
 
-Step 8: Default Username And Password For User Is "shashi" And "shashi"
+Step 8: Default Username And Password For User Is "Shashi" And "Shashi"
```

Comment:
This changes will leads to login failure, because the password and username is case sensitive